nevergrad/optimization/optimizerlib.py

```diff
@@ -977,6 +1016,21 @@ def _internal_tell_not_asked(self, candidate: p.Parameter, loss: tp.FloatLoss) -
             opt.tell(local_candidate, loss)
 
 
+class Rescaled(base.ConfiguredOptimizer):
+    """Configured optimizer for creating rescaled optimization algorithms.
+    """
```

Comment:
Please improve the docstring with the standard format, including the description of parameters (see example for ParametrizedOnePlusOne)
It's important to have a good documentation for users, and the documentation is generated from these docstrings

Reply:
Done.